data/json/monsters.json

```diff
@@ -2531,6 +2540,7 @@
     "armor_bash": 12,
     "armor_cut": 8,
     "path_settings": { "max_dist": 5 },
+    "revert_to_itype": "bot_tribot",
```

Comment:
There is no such item as `bot_tribot` it is called `bot_tripod`.

Reply:
Thanks! Must have gotten lost at some point as I had it working locally, darn internal names.